alldogs.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```

Comment:
There is a meta tag referenced in the week one slides. You'll want to include that.

Reply:
WHOOPS. Fixed.